server/pbenchinacan/load_keycloak.sh

```diff
@@ -17,13 +17,50 @@
 # "http://localhost:8080/*" unless specified otherwise by 'KEYCLOAK_REDIRECT_URI'
 # env variable.
 
+KEYCLOAK_BASE_IMAGE=${KEYCLOAK_BASE_IMAGE:-"images.paas.redhat.com/pbench/pbenchinacan-keycloak:base"}
 KEYCLOAK_HOST_PORT=${KEYCLOAK_HOST_PORT:-"http://localhost:8090"}
+KEYCLOAK_HEALTH_URI="${KEYCLOAK_HOST_PORT}/health"
```

Comment:
Since you "hard-code" the URIs for things like assigning roles, is there any real point to having a variable for the health check URI?  Why not just code it like you've coded the others?

I see that you use it in more than one place which is, perhaps, sufficient justification for symbolizing it.  However, this definition is in the middle of a set which are intended to be overridden by the caller, so you should probably move it down, say, next to where you define `end_in_epoch_secs` (and maybe down-case it, as well).